catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -4,10 +4,10 @@
 
 
 This module can be used to create a command in the following way:
-    poppy_obj = PoppySegmentedCommand(global_coefficients, dm_config_id, laser_config_id)
+    poppy_obj = PoppySegmentedCommand(global_coefficients, dm_config_id, wavelength)
```

Comment:
This would be in all places where `wavelength` is used, but I recommend using `laser_wavelength` as a variable name to make this a little more explicit.